util-jvm/src/main/scala/com/twitter/jvm/Jvm.scala

```diff
@@ -261,7 +261,9 @@ object Jvm {
   private lazy val _jvm =
     try new Hotspot
     catch {
-      case NonFatal(_) => NilJvm
+      case NonFatal(_) | _: UnsatisfiedLinkError =>
+        log.log(Level.WARNING, "failed to create Hotspot JVM interface, using NilJvm instead")
```

Comment:
nit: use `log.warning` instead of `log.log`

Reply:
Thanks for the suggestion @ryanoneill!